### ISSUE TITLE
Skyrim SKSE does not open from MO2

### ISSUE THREAD

**Issue by Hezekiy** [ModOrganizer2/modorganizer#1852]
I've been installing mods for a long time and I've come across several problems, but luckily I managed to solve practically all but one. Before my SKSE opened normally through MO2, but after some time I tried to run the game to see how the performance was and the game simply does not open through MO2. I've already tried manually opening the default SKSE, but the mods don't really work if run with MO2. If anyone can help me with this, I will clear all possible doubts.

(sorry for the bad spelling, I don't know how to speak english)

**Comment by Holt59** [comment]
> but after some time I tried to run the game

If you've not played Skyrim in a long time and are trying SKSE, changes are that Steam updated your game to AE and you need to update SKSE.

**Comment by Hezekiy** [comment]
Sorry, I'm not referring to that. I mean, I always add some mods and then try to run the game when I'm building a pack, that's what I meant. The problem is that it didn't show any bugs before I went to sleep, I just noticed that the game didn't open during the night of the next day (in this case yesterday). I'll try to disable the mods and see if the game opens through MO2 to check the possibility of a conflict.

**Comment by Hezekiy** [comment]
I found out which mod was conflicting but I'm not sure why that conflict, maybe I installed it wrong or it really caused some conflict. Anyway thank you.

if anyone encounters a similar issue, I was loading these mods:
Address Library for SKSE Plugins_backup
Climates Of Tamriel-SE_backup
Immersive Citizens - AI Overhaul_backup
Unmanaged: FNIS
DLC: HearthFires
DLC: Dragonborn
DLC: Dawnguard
Creation Club: ccvsvsse002-pets
Creation Club: ccvsvsse001-winter
Creation Club: cctwbsse001-puzzledungeon
Creation Club: ccqdrsse002-firewood
Creation Club: ccqdrsse001-survivalmode
Creation Club: ccpewsse002-armsofchaos
Creation Club: ccmtysse002-ve
Creation Club: ccmtysse001-knightsofthenine
Creation Club: ccfsvsse001-backpacks
Creation Club: ccffbsse002-crossbowpack
Creation Club: ccffbsse001-imperialdragon
Creation Club: cceejsse004-hall
Creation Club: cceejsse003-hollow
Creation Club: cceejsse002-tower
Creation Club: cceejsse001-hstead
Creation Club: ccedhsse002-splkntset
Creation Club: ccedhsse001-norjewel
Creation Club: ccbgssse064-ba_elven
Creation Club: ccbgssse063-ba_ebony
Creation Club: ccbgssse062-ba_dwarvenmail
Creation Club: ccbgssse060-ba_dragonscale
Creation Club: ccbgssse058-ba_steel
Creation Club: ccbgssse057-ba_stalhrim
Creation Club: ccbgssse051-ba_daedricmail
Creation Club: ccbgssse045-hasedoki
Creation Club: ccbgssse043-crosselv
Creation Club: ccbgssse041-netchleather
Creation Club: ccbgssse040-advobgobs
Creation Club: ccbgssse037-curios
Creation Club: ccbgssse036-petbwolf
Creation Club: ccbgssse035-petnhound
Creation Club: ccbgssse034-mntuni
Creation Club: ccbgssse031-advcyrus
Creation Club: ccbgssse025-advdsgs
Creation Club: ccbgssse021-lordsmail
Creation Club: ccbgssse020-graycowl
Creation Club: ccbgssse019-staffofsheogorath
Creation Club: ccbgssse018-shadowrend
Creation Club: ccbgssse016-umbra
Creation Club: ccbgssse014-spellpack01
Creation Club: ccbgssse013-dawnfang
Creation Club: ccbgssse010-petdwarvenarmoredmudcrab
Creation Club: ccbgssse008-wraithguard
Creation Club: ccbgssse007-chrysamere
Creation Club: ccbgssse006-stendarshammer
Creation Club: ccbgssse004-ruinsedge
Creation Club: ccbgssse003-zombies
Creation Club: ccbgssse002-exoticarrows
TrueHUD - HUD Additions
Traducao em PT-BR do mod TrueHUD - HUD Additions
Nemesis Overwrite
Nemesis
True Directional Movement
True Directional Movement (PTBR)
MCM Helper
Address Library for SKSE Plugins
SkyUI
SkyUI PT-BR
Skyrim 3D Trees and Plants
Skyrim 3D Trees and Plants PT-BR
Abyssal Tides Magic
Abyssal Tides Magic (PTBR)
Soldier of Stendarr
Soldier of Stendarr (PTBR)
Apothecary - An Alchemy Overhaul
Apothecary - An Alchemy Overhaul (PTBR)
Apothecary - Skyrim 3D Trees and Plants Patch
Apothecary - Wyrmstooth Patch
Tempered Skins for Males - Dressed Version
Odin - Skyrim Magic Overhaul
Odin - Skyrim Magic Overhaul 2.0.1 PT-BR
Wyrmstooth
Wyrmstooth PTBR
Tempered Skins for Females CBBE
LC_Citadel
LC_Citadel - Portugues Brasil
Hunterborn
Hunterborn - PORTUGUES BR
Morningstar - Minimalistic Races of Skyrim
Morningstar - Minimalistic Races of Skyrim Traducao PT-BR
Ordinator - Perks of Skyrim
Odin - Ordinator Compatibility Patch
Sacrilege - Minimalistic Vampires of Skyrim
Sacrilege - Minimalistic Vampires of Skyrim Traducao PT-BR
Growl - Werebeasts of Skyrim
Growl - Werebeasts of Skyrim PT-BR
Smilodon - Combat of Skyrim
Smilodon - Combat of Skyrim - Traducao PTBR
Evenstar - Minimalistic Standing Stones of Skyrim
Evenstar - Minimalistic Standing Stones of Skyrim Traducao PT-BR
Unofficial Skyrim Special Edition Patch
Thunderchild - Epic Shouts and Immersion
Thunderchild Portugues Brasileiro
Cutting Room Floor - SSE
Cutting Room Floor - SSE 3.1.11 PT-BR
ENB Helper SE
Picturesque - graphical overhaul - weathers mod plus ENB
powerofthree's Tweaks
DynDOLOD Resources SE
SSE Engine Fixes (skse64 plugin)
Rudy fix for Splashes of Storms and ENB
Splashes Of Storms
Particle Patch for ENB
Moons And Stars - Sky Overhaul SKSE
Less Distracting Blowing Snow Effects for ENB Particle Patch
Enhanced Volumetric Lighting and Shadows (EVLaS)
XP32 Maximum Skeleton Special Extended
Water for ENB
Spell Perk Item Distributor
Bright Waterfall Fix for ENB
Just Blood - Dirt and Blood Lite
Convenient Horses
Convenient Horses PT-BR
Faster Horses
XP32 Maximum Skeleton Special Extended - PTBR
CBPC Equipment Physics
Modern Brawl Bug Fix
Weapons Repositioner SE
Ecotone Dual Sheath
JContainers SE
All Geared Up Derivative SE - AllGUD
Paraphernalia - Magicka
RAID Weathers
Native EditorID Fix
KreatE
LeanWolf's Better-Shaped Weapons - All in One Installer
Kowindy's Reshade For RAID Weather
SETTING SONS ENB
Runed Nord Hero and Anti-Hero Weapons
Make RAID weather horizon more perfect
Real Clouds - Special Edition
ETHEREAL CLOUDS - Special Edition
Vanilla Hair Variety Plus SE
Better Jumping SE
SmoothCam
Dynamic Animation Replacer
BeastHHBB - Khajiit and Argonian content - player character and NPC replacer
SmoothCam - Modern Camera Preset
Skyrim's Paraglider
Skyrim's Paraglider (PTBR)
ENB Light
Immersive Citizens - AI Overhaul
Immersive Citizens - AI Overhaul ptbr
True Storms Special Edition - Thunder Rain and Weather Redone
True Storms Special Edition - Thunder Rain and Weather Redone - PORTUGUES BR
Embers XD
Climates Of Tamriel-SE
The Great City Of Winterhold SSE Edition
The Great City Of Winterhold SSE Edition - PTBR
Climates of Tamriel (PTBR)
Be a Milk Drinker - SPECIAL EDITION
Be a Milk Drinker - SPECIAL EDITION PT BR
Apothecary Food and Drink Patches
Scrambled Bugs

(SkyrimSE v. 1.5.97)
